common/djangoapps/third_party_auth/tests/test_middleware.py

```diff
@@ -0,0 +1,44 @@
+"""
+Test the session-flushing middleware
+"""
+import unittest
+
+from django.conf import settings
+from django.test import Client
+
+
+class TestSessionFlushMiddleware(unittest.TestCase):
+    """
+    Ensure that if the pipeline is exited when it's been quarantined,
+    the entire session is flushed.
+    """
+    @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
```

Comment:
I'm curious: Can you apply this decorator to the whole class rather than each method?